components/nimbus/ios/Nimbus/NimbusBuilder.swift

```diff
@@ -176,9 +176,16 @@ public class NimbusBuilder {
                     onApplyCallback?(nimbus)
                 }
             }
+
+            // Is the app being built locally, and the nimbus-cli
+            // hasn't been used before this run.
+            func isLocalBuild() -> Bool {
+                serverSettings == nil && nimbus.isFetchEnabled()
+            }
+
             if let args = ArgumentProcessor.createCommandLineArgs(args: commandLineArgs) {
                 ArgumentProcessor.initializeTooling(nimbus: nimbus, args: args)
-            } else if let file = initialExperiments, isFirstRun || serverSettings == nil {
```

Comment:
What were the `serverSettings` representing here before?

Reply:
Ostensibly the `RemoteSettings` configuration. It could be `nil` if it wasn't added injected in during the build.